lib/components/user/back-to-trip-planner.js

```diff
@@ -1,14 +1,16 @@
 import React from 'react'
 import styled from 'styled-components'
-import { Link } from 'react-router-dom'
 
+import { Link } from '../form/connected-links'
 import Icon from '../narrative/icon'
 
 const Container = styled.div``
 
 const BackToTripPlanner = () => (
   <Container>
-    <Link to='/'><Icon name='arrow-left' />  Back to trip planner</Link>
+    <Link to={'/'}>
+      <Icon name='arrow-left' />  Back to trip planner
+    </Link>
```

Comment:
I'm a little worried about the style for this component deviating from the very similar `back-link.js`. Is there a way to reuse components?

Reply:
The problem is that the back button is a `<button>` with an event handler, and the link straight `<a>`s, so the only reusable piece is the icon.

Reply:
How about https://github.com/opentripplanner/otp-react-redux/pull/280/commits/6c9e778b2718f014a1a872133a3267371adb4b6f? This commit also updates the icon to have a margin with the link text.